src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -19,7 +19,7 @@
 
 registry_name = CliArgumentType(
     options_list=('--registry-name', '-r'),
-    help='Azure container registry name. A new Azure container registry is created if omitted or does not exist.'
+    help='Azure Container Registry name to which container images will be pushed after each build, and pulled from to deploy container instances. If you have an existing registry at myregistry-{accountname}.azureacr.io, then set this parameter value to myregistry. A new Azure Container Registry is created if this parameter value is omitted or does not exist.'
```

Comment:
A quick warning here. If you generally alias 'registry_name' with the registry_name argument type, then everywhere it occurs you will get this help text. Right now, you will get this help text for both `container release create` and `container build create`. Is that what you want?